gradle/dokka-gradle-example/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    kotlin("jvm") version "1.4.0-M3"
+    id("org.jetbrains.dokka") version ("1.4.0-M3-dev-65")
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven("https://dl.bintray.com/kotlin/kotlin-dev")
+}
+
+dependencies {
+    implementation(kotlin("stdlib"))
+    testImplementation(kotlin("test-junit"))
+}
+
+tasks.withType<DokkaTask> {
+    dokkaSourceSets {
+        configureEach {
```

Comment:
As this is a simple project with one source set, IMHO we can just configure the `main` here

Reply:
Done ✅